Automattic/woocommerce-payments
# Remove UTC label from transaction listing

Fixes #8386 

#### Changes proposed in this Pull Request



This PR removes the `(UTC)` from the `Date / Time` label on the Transactions listing page.

| Before | After |
|--------|--------|
| ![Before](https://github.com/Automattic/woocommerce-payments/assets/57298/42d9286b-0f3b-4e53-bb3b-e4395290a1c8) | ![After](https://github.com/Automattic/woocommerce-payments/assets/57298/97b424d4-a925-402d-ab3e-18660879c410) | 

This label was incorrectly changed in  https://github.com/Automattic/woocommerce-payments/issues/7591. While the label says UTC the time is always rendered in the Merchant's timezone.

I have created #8522 to clarify that CSV's use UTC.

#### Testing instructions

* Go to Payments > Transactions
* See `Date / Time` as column header and times in local time.

-------------------

- [x] Run `npm run changelog` 

**Post merge**
